mypy/semanal.py

```diff
@@ -2674,8 +2674,9 @@ def visit_member_expr(self, expr: MemberExpr) -> None:
                 # one type checker run. If we reported errors here,
                 # the build would terminate after semantic analysis
                 # and we wouldn't be able to report any type errors.
-                full_name = '%s.%s' % (file.fullname() if file is not None else None, expr.name)
-                mod_name = " '%s'" % file.fullname() if file is not None else ''
+                full_name = '%s.%s' % (file.fullname() if file is not None  # type: ignore
+                                       else None, expr.name)
+                mod_name = " '%s'" % file.fullname() if file is not None else ''  # type: ignore
```

Comment:
Why did you add three ``# type: ignore``s in this file? Are you trying to self check mypy with ``--strct-optional``?
If yes, then i think you should not put them here in this PR.